tests_new/integration_tests/core/environment/test_environment.py

```diff
@@ -52,3 +60,60 @@ def test_persistent_env_update(client1, persistent_env1):
     check_stack_in_progress(client1, env_uri, stack_uri)
     stack = check_stack_ready(client1, env_uri, stack_uri)
     assert_that(stack.status).is_equal_to('UPDATE_COMPLETE')
+
+
+def test_invite_group_on_env_no_org(client1, session_env2, group3):
+    assert_that(invite_group_on_env).raises(GqlError).when_called_with(
+        client1, session_env2.environmentUri, group3, ['CREATE_DATASET']
+    ).contains(group3, 'is not a member of the organization')
```

Comment:
I believe `group3` is invited to `org2` which is where `session_env2` is created in 

Should this be `group4` instead - in my testing this does not pass as is

Reply:
You are absolutely right, fixing it on next push.